QUEST_LV_0100.tsv

```diff
@@ -7181,12 +7181,12 @@ QUEST_LV_0100_20150714_007180	{memo X}$Notice/The device is not working due to r
 QUEST_LV_0100_20150714_007181	{memo X}$Notice/Mining Compass is still pointing towards District 4.{nl}Check around District 4 and find a way to remove the rust.#5
 QUEST_LV_0100_20150714_007182	{memo X}$Notice/Magic supply device is fixed.{nl}Try running the Secondary purifier.#5
 QUEST_LV_0100_20150714_007183	$Activating the purifier
-QUEST_LV_0100_20150714_007184	{memo X}Notice/Secondary Purifier is working again.#5
-QUEST_LV_0100_20150714_007185	{memo X}$Notice/Some parts of this purifier are also missing.{nl} Find the parts using Compass Mine.#5
-QUEST_LV_0100_20150714_007186	{memo X}$Notice/The compass is pointing towards District 6.{nl}Search for parts in District 6#5
-QUEST_LV_0100_20150714_007187	{memo X}$Notice /!/Retrieved the missing parts!{nl}Fix the Main Purifier and activate it#5
-QUEST_LV_0100_20150714_007188	{memo X}$Notice/Successfully repaired the Main Purifier!{nl}Main Purifier is now working!
-QUEST_LV_0100_20150714_007189	{memo X}$Notice/Bonfire lighted.#3
+QUEST_LV_0100_20150714_007184	${memo X}$Notice/Secondary Purifier is working again.#5
```

Comment:
I think $ after the {memo X} would be better because of the text extraction scheme we use :wink: